docs/Users_Guide/wrappers.rst

```diff
@@ -210,28 +210,19 @@ METplus Configuration
 | :term:`OBS_ENSEMBLE_STAT_FILE_WINDOW_BEGIN`
 | :term:`OBS_ENSEMBLE_STAT_FILE_WINDOW_END`
 | :term:`ENSEMBLE_STAT_ENS_THRESH`
-| :term:`ENSEMBLE_STAT_ENS_VLD_THRESH`
-| :term:`ENSEMBLE_STAT_ENS_OBS_THRESH`
+| :term:`ENSEMBLE_STAT_VLD_THRESH`
```

Comment:
Since `ENSEMBLE_STAT_ENS_THRESH` and `ENSEMBLE_STAT_VLD_THRESH` set `fcst.ens_thresh` and `fcst.vld_thresh` should they be named `ENSEMBLE_STAT_FCST_ENS_THRESH` and `ENSEMBLE_STAT_FCST_VLD_THRESH` or am I over-thinking this?

Reply:
To be completely accurate, yes, it would make sense to name them including FCST. I made this decision because we don't support them in the obs dictionary. Would it make sense to include them for both fcst and obs?

Reply:
No, they apply to the forecast data that consists of data from N ensemble members. The observations data is a single value... so there's no valid data threshold to be applied. If the obs data is valid, its used. If not, its not.

We COULD technically define them at the highest config file level of context. But the reason we didn't in the Ensemble-Stat config file is that "vld_thresh" is used in multiple contexts. For example, it also appears in the `regrid` dictionary. Defining it at the top-level context could potentially have confusing and unintended consequences.

Reply:
We discussed this and decided that it is OK to leave it as is because these settings are not also supported in the obs dictionary.